docs/conf.py

```diff
@@ -93,6 +93,9 @@
 
         __"""
 
+link_str = f" https://github.com/Qiskit/qiskit-ibm-runtime/blob/{release}/docs/"
+nbsphinx_prolog += link_str + "{{ docname }}"
```

Comment:
```suggestion
vers = release.split(".")
link_str = f" https://github.com/Qiskit/qiskit-ibm-runtime/blob/{vers[0]}.{vers[1]}/docs/"
nbsphinx_prolog += link_str + "{{ docname }}"
```

Reply:
Could you check if the URL is getting formed properly with above change?

Reply:
is the stable branch going to be `0.1` or `0.1.0`?

Reply:
Stable branch will be 0.1